TimberWinR/Codecs/PlainCodec.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json.Linq;
+using TimberWinR.Parser;
+
+namespace TimberWinR.Codecs
+{
+    public class PlainCodec : ICodec
```

Comment:
Plain codec mean that an input line is just text, same as logstash http://logstash.net/docs/1.4.2/codecs/plain a Json codec means that the input line is expected
to be Json